documentation/contribute/add_a_method.qmd

```diff
@@ -3,14 +3,14 @@ title: Add a method
 order: 40
 ---
 
-This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [first setup](first_setup.qmd) pages.
+This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [getting started](getting_started.qmd) pages.
 
 There are 2 type of components you can add:
 
 * `control method`:  The control methods are methods that form a baseline (random/ground truth) to compare the methods from the same task against.
 * `method`: scripts to be benchmarked on how to perform the task.
 
-You can find the components in their respective directories namely the `control_methods` and `methods` directory. The setup of both components is similar with small differences in the config yaml. 
+You can find the components in their respective task directories (`src/task_name/`) namely the `control_methods` and `methods` directory. The setup of both components is similar with small differences in the config yaml. 
 The differences will be pointed out below. whenever `methods` is mentioned it should be replaced with `control_methods` when adding a control method.
 
 ## Directory
```

Comment:
I would structure this section as:

    ## Create a new component
    
    Create a new Viash component by running the following command:
    
    ```bash
    viash run src/common/create_skeleton/config.vsh.yaml -- \
      --task label_projection \
      --method_id my_method \
      -- language python
    ```
    
    This will create a new folder at `src/label_projection/my_method` containing a Viash config (`config.vsh.yaml`) and a script (`script.py` or `script.R`).

Reply:
I was unsure to add this because the `skeleton` component is not finished yet.